src/styles/card.scss

```diff
@@ -0,0 +1,103 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  cursor: pointer;
+  transition: 300ms;
```

Comment:
It's better to specify to what property transition should apply. Fix it in all places.
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/e00b0bec-c0b3-4d8f-a57c-076630443dcf)